### DISCUSSION TITLE
代理设置不要用export http_proxy，会导致前后端无法通信

### DISCUSSION THREAD

**Discussion by luoye2333** [GaiZhenbiao/ChuanhuChatGPT#413] [Show and tell]
如果用export http_proxy设置代理，会导致前后端通信和到openai的通信都设置上代理，导致前后端无法访问。然而如果按照解决方案里的加127.0.0.1的特殊规则设置成直连，那么访问openai的请求也会改成直连，导致无法访问。

也就是说用export http_proxy，要么两者全是代理，要么全不代理。解决方案是在网页端设置代理。

**Comment by luoye2333** [comment]
对了，我用的是v2ray代理，所以访问的是127.0.0.1:10809，才会出现两者不可兼得的情况。
![捕获](https://user-images.githubusercontent.com/32596596/227896131-e7682a36-8bf4-4ba1-8c41-fdc78047c0d4.PNG)
![捕获2](https://user-images.githubusercontent.com/32596596/227896144-ed39a4a3-d197-4a86-a0c3-75224f48ca77.PNG)

**Reply by Git179822868** [reply]
我也是遇到这个情况，在微软云上部署想实现挂梯子
